Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -372,10 +376,12 @@ public void ShowRoundEndScoreboard(string text = "")
                     PlayerOOCName = contentPlayerData?.Name ?? "(IMPOSSIBLE: REGISTERED MIND WITH NO OWNER)",
                     // Character name takes precedence over current entity name
                     PlayerICName = playerIcName,
+                    PlayerGuid = userId?.ToString() ?? "(IMPOSSIBLE: REGISTERED MIND WITH NO OWNER)",
                     PlayerNetEntity = GetNetEntity(entity),
                     Role = antag
                         ? roles.First(role => role.Antagonist).Name
                         : roles.FirstOrDefault().Name ?? Loc.GetString("game-ticker-unknown-role"),
+                    JobRole = roles.FirstOrDefault(role => !role.Antagonist).Name ?? Loc.GetString("game-ticker-unknown-role"),
```

Comment:
If this is intended for indexing by an external service then you should store the actual job prototype ID too. Same with antags. And probably store every antag role instead of just a single one.